### ISSUE TITLE
How to access default subscription

### ISSUE THREAD

**Issue by machwatt** [aldeed/meteor-tabular#81]
First things first:
Awesome integration, fast, easy and smooth. Thanks for the work.

My question is, how would i get the mentioned default subscription, f.e if i want to waitOn in iron:router.
Since my data source is above 10.000 items, it takes a time to load it into the datatables and i want to show custom loading template.

**Comment by kirilan** [comment]
here is how I have done it  this is my router config.js

```
Router.configure({
  layoutTemplate: 'mainLayout',
  loadingTemplate: 'loading',

  yieldTemplates: {
    header: {
      to: 'header'
    },
    footer: {
      to: 'footer'
    }
  },

  onAfterAction: function() {
    $(document).scrollTop(0);
  }
});

Router.onBeforeAction('loading');
```
and then the actual route waits on the subscribtion like this

```
  this.route('dashboard', {
    path: '/dashboard',
    waitOn: function() {
      return this.subscribe("pub1) && this.subscribe("pub2");
    },
    onAfterAction: function() {
      SEO.set({
        title: 'Dashboard | ' + SEO.settings.title
      });
    }
  });
```

hope that's clear enough

**Comment by machwatt** [comment]
The subscription "pub1" and "pub2" are the one published by default from datatables?
As stated here https://github.com/aldeed/meteor-tabular#using-a-custom-publish-function the "package takes care of publication and subscription".

If "pub1" and "pub2" are your own publications via Meteor.publish() then you have two subscriptions on the same data, since this plugin got an own publication?

**Comment by kirilan** [comment]
In this case they are my own publications and they are different
collections so I am not sure how default work if at all

**Comment by machwatt** [comment]
Ok, i dont want to build an own publication method for my 10.000+ items as the datatables integration works well for me. And publish the items twice is quite... slow... 

So i either must implement an own publication method (nooooo.....), or i can maybe reference the handle of the datatables subscription (would be quite nice) :)

**Comment by aldeed** [comment]
Publishing should not be very slow since it only publishes the current page of results. That was really the whole point of creating this package because I needed it for some very large collections.

You may just want to turn on the [processing option](http://datatables.net/reference/option/processing). I don't think waiting on the subs will be possible because of the way it continually rebuilds them.

**Comment by machwatt** [comment]
@aldeed i did not mean the publish method from your plugin. i meant the case, if you use datatables AND an own subscription method (as i did previously...) 

Processing option is nice to have, but its not the problem for datatable to display 10 or 100 items, but the slowest part is getting the subset from 10k items.

waitOn would just be a nice-to-have feature. But thanks for the feedback.
